tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "checkJs": false,
     "declaration": false,
     "target": "es5",
-    "lib": ["es5", "es6", "es2016", "es2019", "es2018.promise", "dom"],
```

Comment:
Why did we remove this? Please check our jsdocs still works after removing.

Reply:
@charliesantos it was removed since I found that es2019 contains everything es2018 and so on and so forth. But how do you double check jsdocs are still working? I've ran `build:quick` and `docs` with no errors but I'm not sure if that's enough.

Reply:
@PikaJoyce after running `build:quick` or `docs`, please check the generated docs under dist folder. Open the index.html file and make sure the docs are rendered properly. You need to check docs generated from a .ts file and .js file.

Reply:
They rendered fine!